xontrib/whole_word_jumping.py

```diff
@@ -1,14 +1,23 @@
-"""Jumping across whole words (non-whitespace) with Ctrl+Left/Right.
+"""Jumping/delete across whole (non-whitespace) words with Ctrl+Left/Right/Delete/Backspace.
+
+Alt+Left/Right/Delete/Backspace remain unmodified to jump/delete over smaller word segments.
+
```

Comment:
can this docstring be simplified ? such as listing the shortcuts and their description

Reply:
Sure. Gave it a try, but let me know if you had something specific in mind and I can resubmit.

I've also ran `black` on the file in an attempt to get the CI tests to pass.

Reply:
Looks good. 

For qa checks, you can use `xonsh run-tests.xsh qa` to check them locally